daemon/indexing/apollo/index.js

```diff
@@ -221,16 +221,17 @@ const resolvers = {
   Query: {
     async listings(root, args, context, info) {
       // TODO: handle pagination (including enforcing MaxResultsPerPage), filters, order.
-      let response = await search.Listing.search(args.searchQuery, args.filters)
+      let {listings, max_price, min_price, listings_total} = await search.Listing
```

Comment:
nit: use camel casing for JS variable names. e.g. maxPrice rather than max_price.
same comments for all the files edited in this PR.

Reply:
Yup thanks for catching that. It shows I was tired when pushing these changes and did too many mistakes.